packages/modules/src/hooks/useContributionsReminderSignup.ts

```diff
@@ -29,6 +30,7 @@ export function useContributionsReminderSignup(
             reminderPlatform,
             reminderComponent,
             reminderStage,
+            reminderOption,
```

Comment:
The api has always accepted this field, we've just not been sending it from the epics & banners before